src/main/java/org/rust/lang/doc/psi/RustDocElementTypes.java

```diff
@@ -0,0 +1,13 @@
+package org.rust.lang.doc.psi;
+
+public interface RustDocElementTypes {
+    RustDocTokenType DOC_TEXT = new RustDocTokenType("<DOC_TEXT>");
+    RustDocTokenType DOC_DECO = new RustDocTokenType("<DOC_DECO>");
```

Comment:
`DECO` comes from _decoration_. It refers to all these `///`, `//!, `/*!`, `*`, etc. Sincerely I don't know how they are properly called in English :sweat_smile: